src/args/arg.rs

```diff
@@ -3311,6 +3312,16 @@ impl<'a, 'b> Arg<'a, 'b> {
         self
     }
 
+    /// Specifies that if the value is not passed in as an argument, that it should be retrieved
+    /// from the environment if available. If it is not present in the environment, then default
+    /// rules will apply.
+    pub fn from_env<K: AsRef<OsStr>>(mut self, name: K) -> Self {
+        self.setb(ArgSettings::TakesValue);
+
+        self.v.from_env = env::var_os(name);
```

Comment:
this needs to be a tuple of OsStr and the env::var, I'll be adding that in a subsequent change.